fuse/dl/lightning/pl_funcs.py

```diff
@@ -137,7 +136,7 @@ def model_default_callbacks(
             callbacks.append(model_checkpoint_display)
 
     # last epoch checkpoint
-    callbacks.append(ModelCheckpoint(dirpath=model_dir, filename="last_epoch", save_last=True))
+    callbacks.append(ModelCheckpoint(dirpath=model_dir, save_last=True, save_top_k=0))
```

Comment:
save_top_k=1 could be a better default

Reply:
Why?
(We use several callbacks, one to save the last checkpoint and others to monitor metrics/losses)